infrastructure/persistence/thirdparty/youtube.go

```diff
@@ -0,0 +1,9 @@
+package thirdparty
```

Comment:
The thirdparty directory seems unnecessary, and it seems odd that it is a subdirectory of the persistence directory as well. (Because persistence is only a place to put the values ​​stored in the database. If you consider putting it in redis, it would be nicer to design the response received from youtube.go to be saved in redis.go.)

Instead, you can move the youtube-related logic to /infrastructure/client/youtube.go.

Reply:
One thing I would like to suggest is that merge this mr first and then modify the code related to issue above in the mr I am working on.

For reference, I'm currently making a youtube v3 client and when I push a merge request, I'll move the code included here into the client/youtube.go.

Reply:
@minkj1992 
Oh, I thought it would be ambiguous to put this part under persistence, but that's a good idea !!
I am referring to the figure below, and I will think of the 3 party APIs as the `/infrastructure/client/**.go`

![hexagonal-architecture](https://user-images.githubusercontent.com/47515936/180781078-a94e37d1-82bc-4d0e-8373-977d1e458db2.png)

Reply:
> One thing I would like to suggest is that merge this mr first and then modify the code related to issue above in the mr I am working on.
>
> FYI, I'm currently making a youtube v3 client and when I push a merge request, I'll move the code included here into the client/youtube.go. Is that okay?

okay good ! 👍